PyFunceble/cli/__init__.py

```diff
@@ -252,7 +253,7 @@ def tool():  # pragma: no cover pylint: disable=too-many-branches,too-many-state
                     "--domain",
                     type=str,
                     nargs="+",
-                    help="Set and test the given domain.",
+                    help="Test the given domain(s). Multiple space separated domains can be given.",
```

Comment:
s/Test the given domain(s). Multiple space separated domains can be given./Test one or more domains, separated by spaces./

Reply:
I used the `Multiple space separated XYZ can be given.`  as general formulation (see for example `--dns`) :thinking:

Reply:
Then you have to weight the suggestion against your feeling for a shorter line or leave it as is :smile: 

But `--dns` could easily be altered to 

`Set the DNS server(s) we have to work with. Multiple space separated DNS server can be given.`
=>
`Set specific DNS serves to be used doing this test, separated by spaces`

Reply:
Or maybe just
`Set specific DNS serves to use doing this test, separated by spaces`